expo-maps/ios/ExpoMaps/AppleMaps/AppleMapsDelegate.swift

```diff
@@ -143,41 +150,72 @@ class AppleMapsDelegate : NSObject, MKMapViewDelegate {
         return clusterAnnotation
       }
     }
-
+    
     return ExpoMKClusterColorAnnotation(memberAnnotations: memberAnnotations)
   }
   
+  func mapViewDidChangeVisibleRegion(_ mapView: MKMapView) {
+    appleMapsView?.onRegionChange(
+        CameraPositionRecord(camera: mapView.camera, coordinateSpan: mapView.region.span).toDictionary()
+    )
+  }
+  
   func mapView(_ mapView: MKMapView, regionWillChangeAnimated animated: Bool) {
-    let position = mapView.camera.centerCoordinate
-    sendEvent(MapEventsNames.ON_CAMERA_MOVE_STARTED_EVENT.rawValue, createCameraEventContent(latitude: position.latitude, longitude: position.longitude))
+    appleMapsView?.onRegionChangeStarted(
+        CameraPositionRecord(camera: mapView.camera, coordinateSpan: mapView.region.span).toDictionary()
+    )
   }
   
   func mapView(_ mapView: MKMapView, regionDidChangeAnimated animated: Bool) {
-    let position = mapView.camera.centerCoordinate
-    sendEvent(MapEventsNames.ON_CAMERA_MOVE_ENDED_EVENT.rawValue, createCameraEventContent(latitude: position.latitude, longitude: position.longitude))
+    appleMapsView?.onRegionChangeComplete(
+        CameraPositionRecord(camera: mapView.camera, coordinateSpan: mapView.region.span).toDictionary()
+    )
+  }
+  
+  func mapViewDidFinishRenderingMap(_ mapView: MKMapView, fullyRendered: Bool) {
+    if !finishedInitialLoad {
+      appleMapsView?.onMapLoaded("")
```

Comment:
```suggestion
      appleMapsView?.onMapLoaded(null)
```
It would be better to pass null instead of empty string